app/code/BodenkoVV/AskQuestion/Controller/Adminhtml/Question/Edit.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace BodenkoVV\AskQuestion\Controller\Adminhtml\Question;
+
+use Magento\Backend\App\Action;
+
+/**
+ * Class Edit
+ * @package BodenkoVV\AskQuestion\Controller\Adminhtml\Question
+ */
+class Edit extends Action
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'BodenkoVV_AskQuestion::save';
+
+    /**
+     * Core registry
+     *
+     * @var \Magento\Framework\Registry
+     */
+    protected $_coreRegistry;
+
+    /**
+     * @var \Magento\Framework\View\Result\PageFactory
+     */
+    protected $resultPageFactory;
+
+    /** @var string $_pageName */
+    protected $_pageName='Question Edit Item';
+
+    /**
+     * @param Action\Context $context
+     * @param \Magento\Framework\View\Result\PageFactory $resultPageFactory
+     * @param \Magento\Framework\Registry $registry
+     */
+    public function __construct(
+        Action\Context $context,
+        \Magento\Framework\View\Result\PageFactory $resultPageFactory,
+        \Magento\Framework\Registry $registry
+    ) {
+        $this->resultPageFactory = $resultPageFactory;
+        $this->_coreRegistry = $registry;
+        parent::__construct($context);
+    }
+
+    /**
+     * Init actions
+     *
+     * @return \Magento\Backend\Model\View\Result\Page
+     */
+    protected function _initAction()
+    {
+        // load layout, set active menu and breadcrumbs
+        /** @var \Magento\Backend\Model\View\Result\Page $resultPage */
+
+        $resultPage = $this->resultPageFactory->create();
+        $resultPage->addBreadcrumb(__($this->_pageName), __($this->_pageName));
+        $resultPage->getConfig()->getTitle()->prepend(__($this->_pageName));
+        return $resultPage;
+    }
+
+    /**
+     * Edit CMS page
```

Comment:
Seems you've stolen this code in CMS module, 
please remove unnecessary things of CMS page editing, which are not related to Questions grid, and rename the rest of the methods

Reply:
+